src/main/java/org/apache/commons/imaging/formats/tiff/TiffImageData.java

```diff
@@ -140,8 +145,8 @@ public ImageDataReader getDataReader(final TiffDirectory directory,
     public abstract ImageDataReader getDataReader(TiffDirectory directory,
             PhotometricInterpreter photometricInterpreter, int bitsPerPixel,
             int[] bitsPerSample, int predictor, int samplesPerPixel, int width,
-            int height, int compression, ByteOrder byteOrder) throws IOException,
-            ImageReadException;
+      int height, int compression, int planarConfiguration,
+      ByteOrder byteOrder) throws IOException,            ImageReadException;
```

Comment:
Not sure if this is GitHub's UI issue, but looks like there are several spaces between `IOException, ` and `ImageReadException`?

Reply:
Some of the spacing got accidentally modified when I edited file.  I thought I cleaned them all, but I missed this one. Sorry.